wetterdienst/dwd/radar/metadata.py

```diff
@@ -0,0 +1,111 @@
+from enum import Enum
+
+
+class RadarDate(Enum):
+
+    LATEST = "latest"
+    CURRENT = "current"
+    MOST_RECENT = "most_recent"
+
+
+class RadarDataFormat(Enum):
+    """
+    Enumeration for different radar data formats.
+    """
+
+    BINARY = "binary"
+    BUFR = "bufr"
+    HDF5 = "hdf5"
+
+
+class RadarDataSubset(Enum):
+    """
+    Enumerate HDF5 subset types for radar sweep data.
+
+    https://opendata.dwd.de/weather/radar/sites/sweep_pcp_v/boo/hdf5/
+    """
+
+    SIMPLE = "simple"
+    POLARIMETRIC = "polarimetric"
+
+
+class RadarParameter(Enum):
+
+    # Composites
+    # https://docs.wradlib.org/en/stable/notebooks/fileio/wradlib_radar_formats.html#German-Weather-Service:-RADOLAN-(quantitative)-composit  # noqa:E501,B950
+
+    # https://opendata.dwd.de/weather/radar/composit/
+    # TODO: "FX" was missing here. OTOH, "pp" doesn't
+    #       seem to be present on opendata.dwd.de.
+    FX_REFLECTIVITY = "fx"
+    PG_REFLECTIVITY = "pg"
+    # PP_REFLECTIVITY = "pp"
```

Comment:
Dear @meteoDaniel,

I found `PP_REFLECTIVITY` here but I haven't been able to find it neither at https://opendata.dwd.de/weather/radar/composit/ nor elsewhere on the public space of the DWD data repository.

Maybe it was just a typo? Please advise.

With kind regards,
Andreas.

Reply:
https://www.dwd.de/DE/leistungen/radarprodukte/radarkomposit_pp.pdf?__blob=publicationFile&v=2 

There are 4 composite types. I guess this document is outdated.

Reply:
Thanks. So, shall we just remove it from the code base?

Reply:
Removed through 204830b6b58.